x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/components/view_results_panel/view_results_panel.tsx

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC, Fragment } from 'react';
+import { EuiCard, EuiIcon } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { useNavigateToPath } from '../../../../../contexts/kibana';
+import { getResultsUrl } from '../../../analytics_management/components/analytics_list/common';
+import { ANALYSIS_CONFIG_TYPE } from '../../../../common/analytics';
+
+interface Props {
+  jobId: string;
+  analysisType: ANALYSIS_CONFIG_TYPE;
+}
+
+export const ViewResultsPanel: FC<Props> = ({ jobId, analysisType }) => {
+  const navigateToPath = useNavigateToPath();
+
+  const redirectToAnalyticsManagementPage = async () => {
+    const path = getResultsUrl(jobId, analysisType);
+    await navigateToPath(path);
+  };
+
+  return (
+    <Fragment>
+      <EuiCard
+        // @ts-ignore
```

Comment:
nit: would be good to comment why we need this ts-ignore

Reply:
Moved the inline style to a separate file so this ignore isn't needed anymore. d12ebe435567047801b5da68847d3a54a520a0c5